src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -3746,6 +3746,73 @@ protected SELF satisfiesExactlyInAnyOrderForProxy(Consumer<? super ELEMENT>[] re
     return myself;
   }
 
+  /**
+   * Verifies that there is exactly one element of the iterable elements that satisfies the {@link Consumer}.
+   *
```

Comment:
add a paragraph to separate the examples from the description, a line like should be enough:
```
  * <p>
```